Dockerfile

```diff
@@ -1,26 +1,29 @@
-FROM node:8-alpine
+FROM node:12-alpine
 MAINTAINER "Dan Farrelly <daniel.j.farrelly@gmail.com>"
 
 ENV NODE_ENV production
 
-RUN apk add --no-cache curl
+WORKDIR /root
 
-RUN mkdir -p /usr/src/app
-WORKDIR /usr/src/app
+COPY package* .
```

Comment:
This fails with the error:
> When using COPY with more than one source file, the destination must be a directory and end with a /

_Docker version 19.03.5_

Can you update this to instead copy to `/root/`?